r/rsparkling/R/package.R

```diff
@@ -2,13 +2,25 @@
 #' @importFrom sparklyr spark_dependency register_extension invoke_static invoke spark_connection spark_dataframe sdf_register spark_context
 #' @importFrom h2o h2o.getFrame h2o.getId h2o.init
 
+#' @import sparklyr h2o utils
```

Comment:
Instead of importing whole namespace use `#' @importFrom pkg fun`, we don't want to get into namespace collisions. When changing imports you also need to render NAMESPACE each time, otherwise it won't take effect.